backend/apps/views/__init__.py

```diff
@@ -0,0 +1 @@
+from .organizations import *
```

Comment:
A more general comment, but I would see to avoid omitting a newline character at the end of files. Just always add one.

This is more due to some tools having issues processing files, since some of them rely on a newline character to determine the end of the file. Does not always happen, but it can, which is why GitHub usually adds this warning.

Reply:
Ya I think at one point we need to do a sit down for the dev community that's focussed on what tools we're using in our environment. Specifically we all should be using the same formatting process, and this process should enforce having a line at the end of the file on save. TBH I have not consciously done this in years, and if I didn't have an extension doing it for me I probably would forget :) What it is that we agree on can then be added to [STYLEGUIDE.md](https://github.com/activist-org/activist/blob/main/STYLEGUIDE.md) as well as [.vscode/extensions.json](https://github.com/activist-org/activist/blob/main/.vscode/extensions.json). This wouldn't be a full dev sync, but maybe 15 min in one where people are told to prepare before. Maybe the one after next as the next one will be quite busy with backend again?

For the above an idea on reinforcing this would be to have another workflow similar to the [maintainer checklist](https://github.com/activist-org/activist/pull/209#issuecomment-1620588231) that would fire on first PRs to the repo. We could shift the check of the commit messages to this as it really just needs to be done once, and beyond that we could have a point that directs people to the style guide and links to the extensions that we as a community think they should be using.

Plus we can all take a look at @momanyisamuel's VS Code setup and marvel at his sidebar on the left 🤪😅😊